Generator/FCBBarCodeGenerator.swift

```diff
@@ -15,7 +15,7 @@ struct FCBBarCodeGenerator {
     
     // MARK: Public Methods
     
-    func barcodeWithCode(code: String, type: FCBBarcodeType, size: CGSize) -> UIImage? {
+    func barcodeWithCode(_ code: String, type: FCBBarcodeType, size: CGSize) -> UIImage? {
```

Comment:
@ezefranca would't be more "swiftier" if this method name would be changed to
`func barcode(with code: String, type: FCBBarcodeType, size: CGSize) -> UIImage?`